emulation/httpserver/routes/allergy/allergens.js

```
@@ -0,0 +1,19 @@
+'use strict';
+
+const LexiconService = require('../../../../services/lexiconService');
+
+const lookupAllergens = (req, res) => {
+    // Lookup Allergens by search term, e.g. "CHOCOLATE"
+    const searchTerm = req.query.search || '';
+
+    LexiconService.lookupAllergies(searchTerm, (err, result) => {
+        if (err) console.log('err: ', err);
```

So for now, if the service function returns an error, let's just return a 500 error code with the Error object returned as JSON:
```Javascript
if (err) return res.status(500).json(err);
```